adapter/internal/oasparser/envoyconf/routes_with_clusters.go

```diff
@@ -215,6 +216,38 @@ func CreateRoutesWithClusters(mgwSwagger model.MgwSwagger, upstreamCerts map[str
 			routes = append(routes, routesP...)
 		}
 		return routes, clusters, endpoints, nil
+
+	}
+
+	if mgwSwagger.EndpointType == constants.AwsLambda {
+		for _, resource := range mgwSwagger.GetResources() {
+			amznResourceName := ""
+			for i, operation := range resource.GetOperations() {
+				value := model.ResolveAmznResourceName(operation.GetVendorExtensions())
+
+				if (i != 0) && (amznResourceName != value) {
+					logger.LoggerOasparser.Errorf("All ARNs must be the same value.")
+				} else if value == "" {
+					logger.LoggerOasparser.Errorf("ARN cannot be empty.")
+				} else {
+					amznResourceName = value
+				}
+				resource.SetAmznResourceName(amznResourceName)
+			}
+			//TODO: interceptor ep
```

Comment:
Shall we remove this to-do? Let's check AWS lambda feature works with interceptors https://apim.docs.wso2.com/en/latest/deploy-and-publish/deploy-on-gateway/choreo-connect/message-transformation/message-transformation-overview/.

Reply:
Fixed. [https://github.com/wso2/product-microgateway/pull/3278/commits/c22ee2ce8f5f8cdb7ce92bfbe25e3bc85e5d0a96](https://github.com/wso2/product-microgateway/pull/3278/commits/c22ee2ce8f5f8cdb7ce92bfbe25e3bc85e5d0a96)